src/app/components/data-entry-grid/index.html

```diff
@@ -117,6 +117,11 @@ <h2>ag-Grid wrapper component</h2>
         The <sky-code>sky-ag-grid-wrapper</sky-code> component provides WCAG-compliant keyboard navigation and sticky column headers for grids that use auto height. Use the <sky-code>sky-ag-grid-wrapper</sky-code>  component to wrap all instances of <sky-code>ag-grid-angular</sky-code>. The wrapper does not constrain standard ag-Grid functionality.
       </p>
 
+      <h2>Data manager directive</h2>
+      <p>
+        If you're using an ag-Grid within the <a routerLink="../data-manager">data manager component</a> there is a directive <sky-code>skyAgGridDataManagerAdapter</sky-code> that will manage some of the interactions between the grid and the data manager service that are typically the same across grid uses. The directive can be added to the <sky-code>sky-data-view</sky-code> element containing the grid. It will initialize and keep up to date the columns listed as visible in the data state, update the data state when the columns are moved or changed within the grid, initialize and keep up to date the selected rows from the data state, update the data state when row selection changes within the grid, and update the data state when the sort is changed within the grid.
```

Comment:
From a technical aspect, this seems good. Only thing I'd ask is there a better way to present the final sentence `It will initialize and keep up to date...` ? The docs team might have some thoughts there...

Reply:
yeah I don't love that but not sure how to make it better. Maybe keep in sync with the data manager state instead of keep up to date?

Reply:
Resolving this conversation as the wording was updated.